dom.d.ts

```diff
@@ -0,0 +1,46 @@
+type Nil = null | undefined;
+type NonNilPrim = string | number | bigint | boolean;
+type Prim = Nil | NonNilPrim;
+
+interface Stringable {
+    [Symbol.toStringTag](): string
```

Comment:
[`Symbol.toStringTag`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol/toStringTag) defines a getter, not a method.

The actual definition of `isStringable` in Prax relies on having a custom `toString`. Using `Symbol.toStringTag` as a "hack" might work for a large number of built-ins. But this would result in false positives for user-defined classes with a custom `toString`, but without a custom `Symbol.toStringTag`. I'm not sure if there's a better way, though.